src/main/scala/slog/Logger.scala

```diff
@@ -0,0 +1,13 @@
+package slog
+
+import slog.macros.LoggerImpl
+
+class Logger[F[_]](val underlying: LocationAwareLogger[F]) {
```

Comment:
I suppose the `LocationAwareLogger` is a stand in for `slf4j` `Logger`, is that correct?

Reply:
Not really. It's actual logger implementation that doesn't need any macros.